src/content/docs/cypher/query-clauses/call.md

```diff
@@ -100,20 +99,15 @@ CALL show_tables() RETURN *;
 ```
 Output:
 ```
--------------------------------------------------------
-|id     | name        | type    | comment             |
-|UINT64 | STRING      | STRING  | STIRNG              |
--------------------------------------------------------
-|0      | gf          | RDF     |                     |
--------------------------------------------------------
-|1      | gf_TRIPLES  | REL     |                     |
--------------------------------------------------------
-|2      | gf_RESOURCE | NODE    |                     |
--------------------------------------------------------
-|3      | person      | NODE    | person info         |
--------------------------------------------------------
-|4      | knows       | REL     | person knows person |
--------------------------------------------------------
+┌─────────────┬──────────────┬────────┬─────────────────────────┐
+│ id          │ name         │ type   │ Comment                 │
+│ INT32       │ STRING       │ STRING │ STRING                  │
+├─────────────┼──────────────┼────────┼─────────────────────────┤
+│ 0           │ gf_TRIPLES   │ REL    │                         │
```

Comment:
Could you rename these two to be something else? `gf_TRIPLES` is still related to RDF graphs, I believe.